src/doc/trpl/closures.md

```diff
@@ -44,14 +45,14 @@ let plus_one = |x: i32| -> i32 { x + 1 };
 assert_eq!(2, plus_one(1));
 ```
 
-But we don’t have to. Why is this? Basically, it was chosen for ergonomic reasons.
-While specifying the full type for named functions is helpful with things like
-documentation and type inference, the types of closures are rarely documented
-since they’re anonymous, and they don’t cause the kinds of error-at-a-distance
-problems that inferring named function types can.
+But we don’t have to. Why is this? Basically, it was chosen for ergonomic
+reasons.  While specifying the full type for named functions is helpful with
+things like documentation and type inference, types within closures are rarely
```

Comment:
Ahh, I see what you mean there.. maybe it would be better to use one of these?

- types in the context of closures
- in the context of closures, types

any thoughts?